// Code derived from https://github.com/broofa/node-uuid
// Reference: https://github.com/broofa/node-uuid/commit/616ad3800f35cf58089215f420db9654801a5a02

// https://github.com/broofa/node-uuid/blob/master/LICENSE.md
/*

The MIT License (MIT)

Copyright (c) 2010-2012 Robert Kieffer

Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

*/

diff --git a/uuid.js b/uuid.js
index 0b596fc..8c1a26b 100644
--- a/uuid.js
+++ b/uuid.js
@@ -17,7 +17,7 @@
   // Node.js crypto-based RNG - http://nodejs.org/docs/v0.6.2/api/crypto.html
   //
   // Moderately fast, high quality
-  if (typeof(_global.require) == 'function') {
+  if ('function' === typeof require) {
     try {
       var _rb = _global.require('crypto').randomBytes;
       _rng = _rb && function() {return _rb(16);};
